wdn/templates_4.1/less/layouts/noscript.less

```diff
@@ -24,7 +24,6 @@
     background-color: @warning;
     color: #000000;
     .sans-serif-font();
-    font-weight: bold;
```

Comment:
No that it matters for this pull request, but it should be documented that this base font _does not_ include a bold weight. The chances for people to accidentally use it are going to be common.